src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -9110,9 +9110,9 @@ public void M()
         }
 
         [Fact]
-        public void AnonymousTypeTemplateSymbolNeverSkipsLocalsInit()
+        public void AnonymousTypeTemplateSymbolDelegatesToModuleWhenSkipLocalsInitAttributeIsNotFound()
```

Comment:
What does `...DelegatesToModuleWhenSkipLocalsInitAttributeIsNotFound` mean? Aren't we always delegating to the module for anonymous type methods?

Reply:
I kept the pattern from `SourceMemberMethodSymbolDelegatesToTypeWhenSkipLocalsInitAttributeIsNotFound` and didn't realized it doesn't make sense. I'll fix it.